config/default.schema.json

```diff
@@ -22,6 +22,11 @@
 			"default": "example",
 			"description": "The API Key that the Sync Cron-Job uses to authenticate itself.This should be removed with SC-3387"
 		},
+		"SILENT_ERROR_ENABLED": {
```

Comment:
this has been added already in https://github.com/hpi-schul-cloud/schulcloud-server/commit/33021c55352580bbcf5f8b19048c91de63eb71f4 see https://github.com/hpi-schul-cloud/schulcloud-server/blob/fc1278f6648721b58e2ac3ebe10f24aa59a17733/config/default.schema.json#L369